src/nft/components/collection/Activity.tsx

```diff
@@ -116,17 +125,21 @@ export const Activity = ({ contractAddress, rarityVerified, collectionName, chai
   const Filter = useCallback(
     function ActivityFilter({ eventType }: { eventType: ActivityEventType }) {
       const isActive = activeFilters[eventType]
+      const activeBackgroundColor = isDarkMode ? vars.color.gray500 : vars.color.gray200
```

Comment:
nit: could memoize this

Reply:
no need its already in a callback with isDarkMode as a dependency